docs/en/sql-reference/data-types/enum.md

```diff
@@ -164,3 +164,5 @@ Enum values are also convertible to numeric types using the `toT` function, wher
 The Enum type can be changed without cost using ALTER, if only the set of values is changed. It is possible to both add and remove members of the Enum using ALTER (removing is safe only if the removed value has never been used in the table). As a safeguard, changing the numeric value of a previously defined Enum member will throw an exception.
 
 Using ALTER, it is possible to change an Enum8 to an Enum16 or vice versa, just like changing an Int8 to Int16.
+
+ClickHouse does not validate Enum values on insertion — it allows inserting numeric values outside the defined Enum mapping
```

Comment:
`nit:` Is this important enough to be a `note` instead of just being an extra line? Perhaps if many people run into this, might make sense for this to be more explicit.